ui/src/components/UserProfile/UserProfile.css

```diff
@@ -20,3 +20,8 @@
 .favorites {
```

Comment:
fyi: when you are applying the same styling to mulitple classes you can do something like 
```
.favorites, .following { 
     font-weight: bold;
}

Reply:
Good to know! Thank you!